src/qtism/data/storage/xml/marshalling/MatchInteractionMarshaller.php

```diff
@@ -118,11 +137,16 @@ protected function marshallChildrenKnown(QtiComponent $component, array $element
 
         // prompt.
         if ($component->hasPrompt() === true) {
-            $element->appendChild($this->getMarshallerFactory()->createMarshaller($component->getPrompt())->marshall($component->getPrompt()));
+            $element->appendChild(
+                $this->getMarshallerFactory()->createMarshaller($component->getPrompt())->marshall(
+                    $component->getPrompt()
+                )
+            );
         }
 
         // shuffle.
-        if ((Version::compare($version, '2.1.0', '>=') === true && $component->mustShuffle() !== false) || Version::compare($version, '2.1.0', '<') === true) {
+        if ((Version::compare($version, '2.1.0', '>=') === true && $component->mustShuffle(
+                ) !== false) || Version::compare($version, '2.1.0', '<') === true) {
```

Comment:
could you provide here proper multiline if statement as suggested earlier or revert that place